include/alpaka/acc/AccGpuUniformCudaHipRt.hpp

```diff
@@ -64,7 +65,12 @@ namespace alpaka
         public BlockSyncUniformCudaHipBuiltIn,
         public IntrinsicUniformCudaHipBuiltIn,
         public MemFenceUniformCudaHipBuiltIn,
-        public rand::RandUniformCudaHipRand<TApi>,
+#    ifdef ALPAKA_DISABLE_VENDOR_RNG
+        public rand::RandDefault,
+#    else
+          public rand::RandUniformCudaHipRand<TApi>
+        ,
```

Comment:
This looks odd. Did `clang-format` do this?

Reply:
Yes... 
I changed the other files and moved the comma in front to avoid the ugly code, but I missed this one.

Reply:
Actually, maybe I did not push that change? I'll check.

Reply:
I missed a couple files, hopefully they look better now.